src/main/java/edu/iis/mto/time/Order.java

```diff
@@ -7,62 +7,75 @@
 import org.joda.time.Hours;
 
 public class Order {
-	private static final int VALID_PERIOD_HOURS = 24;
-	private State orderState;
-	private List<OrderItem> items = new ArrayList<OrderItem>();
-	private DateTime subbmitionDate;
-
-	public Order() {
-		orderState = State.CREATED;
-	}
-
-	public void addItem(OrderItem item) {
-		requireState(State.CREATED, State.SUBMITTED);
-
-		items.add(item);
-		orderState = State.CREATED;
-
-	}
-
-	public void submit() {
-		requireState(State.CREATED);
-
-		orderState = State.SUBMITTED;
-		subbmitionDate = new DateTime();
-
-	}
-
-	public void confirm() {
-		requireState(State.SUBMITTED);
-		int hoursElapsedAfterSubmittion = Hours.hoursBetween(subbmitionDate, new DateTime()).getHours();
-		if(hoursElapsedAfterSubmittion > VALID_PERIOD_HOURS){
-			orderState = State.CANCELLED;
-			throw new OrderExpiredException();
-		}
-	}
-
-	public void realize() {
-		requireState(State.CONFIRMED);
-		orderState = State.REALIZED;
-	}
-
-	State getOrderState() {
-		return orderState;
-	}
-	
-	private void requireState(State... allowedStates) {
-		for (State allowedState : allowedStates) {
-			if (orderState == allowedState)
-				return;
-		}
-
-		throw new OrderStateException("order should be in state "
-				+ allowedStates + " to perform required  operation, but is in "
-				+ orderState);
-
-	}
-
-	public static enum State {
-		CREATED, SUBMITTED, CONFIRMED, REALIZED, CANCELLED
-	}
+
+    public static final int VALID_PERIOD_HOURS = 24;
+    private State orderState;
+    private List<OrderItem> items = new ArrayList<>();
+    private DateTime subbmitionDate;
+
+    public Order() {
+        orderState = State.CREATED;
+    }
+
+    public void addItem(OrderItem item) {
+        requireState(State.CREATED, State.SUBMITTED);
+
+        items.add(item);
+        orderState = State.CREATED;
+
+    }
+
+    public void submit(DateTime submissionDate) {
```

Comment:
to nie jest dobre rozwiązanie kiedy parametr metody jest tak na prawdę wskaźnikiem sposobu uruchomienia - gdyby nie było testów to byłby zbędny